.github/workflows/release-build-outputs.yml

```diff
@@ -0,0 +1,163 @@
+# HMS Networks Solution Center
+# Build Outputs Action for Maven-based Ewon ETK Project Releases
+# Version: 1.0
+#
+# This action is configured to automatically run when a release
+# tag is created in the following syntax: `v*`.
+
+name: Release (Build Outputs)
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  CHANGELOG_FILE_NAME: CHANGELOG.md
+  RELEASE_INCLUDED: CHANGELOG.md README.md LICENSE src .classpath .project pom.xml build.xml
```

Comment:
Are there other files/folders that need to be included?

Reply:
with this PR the release is being synced up with the other connector's releases. The connector specific connector config web page is now dormant and no longer needed in the release. It will stay on the codebase, but is removed from the release here and removed from documentation in the next PR.